### PR TITLE
Renamed and aligned admin kubeconfig

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue [hunk 1]
```diff
@@ -8,39 +8,41 @@ SPDX-License-Identifier: Apache-2.0
...
-        </div>
+        Kubeconfig - Time-Limited Access
+        <template #description>
+          <div class="wrap-text d-flex">
+            <span class="mr-2">
+              Request a kubeconfig valid for
+            </span>
+            <g-popover
+              v-model="popover"
+              toolbar-title="Configure Admin Kubeconfig Lifetime"
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue:19]: Shouldn't we also rename this to e.g. `Configure Kubeconfig Lifetime`

File: frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue [hunk 2]
```diff
@@ -8,39 +8,41 @@ SPDX-License-Identifier: Apache-2.0
...
+                  color="primary"
+                  variant="tonal"
+                  class="pointer"
+                  v-bind="props"
+                >
+                  {{ expirationTitle }}
+                </v-chip>
+              </template>
+              <div class="pa-2">
+                You can configure the <span class="font-weight-bold">kubeconfig lifetime</span> on the
```
Comment (holgerkoser) [inline frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue:36]: I would remove the bold formating here. What do you think?

File: frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue (hunk 1 above)
Comment (petersutter) [reply frontend/src/components/ShootDetails/GShootAdminKubeconfig.vue:19]: Configure Kubeconfig Lifetime +1

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**